azure-pipelines.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         inputs:
           inputs:
           filePath: yarn-ci.sh
```

Comment:
FYI: `yarn-ci.sh` runs `midgard-yarn` (it's a different package)

Reply:
can you point me to docs explaining why is that custom package needed in first place? thx

Reply:
It's an internal fork Yarn v1 (https://github.com/VincentBailly/yarn) that has perf optimizations that's due a policy of [Yarn maintainers](https://github.com/yarnpkg/yarn/issues/7921#issuecomment-753416676) were never merged. It's used in many products internally as has significantly better performance (3-5 times difference for `yarn install`). I tagged you on an internal thread.

Reply:
updated !